store/helper/helper_test.go

```diff
@@ -61,13 +62,13 @@ func (s *mockStore) TLSConfig() *tls.Config {
 }
 
 func (s *HelperTestSuite) SetUpSuite(c *C) {
-	go s.mockPDHTTPServer(c)
+	url := s.mockPDHTTPServer(c)
 	time.Sleep(100 * time.Millisecond)
 	mvccStore := mocktikv.MustNewMVCCStore()
 	mockTikvStore, err := mockstore.NewMockTikvStore(mockstore.WithMVCCStore(mvccStore))
 	s.store = &mockStore{
 		mockTikvStore.(tikv.Storage),
-		[]string{"127.0.0.1:10100/"},
+		[]string{url[len("http://"):]},
```

Comment:
would it use the port that already in use?

Reply:
No, `httptest.NewServer` will binds the socket to port 0, which the kernel will assign it a free port.

Reply:
Got it, thanks.